docker-compose.yml

```diff
@@ -36,6 +34,12 @@ services:
     build:
       context: ./webui
       dockerfile: Dockerfile
+    develop:
+      watch:
+        - path: ./webui
+          ignore:
+            - ./webui/**/tests/**
+          action: rebuild
```

Comment:
@LukeLalor how's this? I'm trying to not rebuild when we have changes in the tests

Reply:
Fixed it. Thanks 🫡